recipes/libsequence/build.sh

```diff
@@ -2,5 +2,7 @@
 
 CPPFLAGS="-I$PREFIX/include $CPPFLAGS" LDFLAGS="-L$PREFIX/lib $LDFLAGS" ./configure --prefix=$PREFIX
 make
+export LD_LIBRARY_PATH=-L$HOME/$PREFIX/lib
```

Comment:
@bgruening Even more confusing: recipes/libsequence/build.sh does not contain the lines that you are referring to.  Can you clarify what you're after here?  I updated this in fwdpp, though...